analyses/molecular-subtyping-EWS/ews-subset/EWS_subtyping_path_dx_strings.json

```diff
@@ -0,0 +1,17 @@
+{
+    "exact_path_dx": [
+        "Ewings Sarcoma"
+    ],
+    "path_free_text_exact": [
+        "ewing's sarcoma",
+        "Ewings Sarcoma;Metastatic secondary tumors",
```

Comment:
check these against the path dx. If respective path dx of these free text dx are being captured as EWS, then we do not need to add these.